### PR TITLE
Update training script

### PR DESCRIPTION
**Pull request by MateoLostanlen**
The purpose of this PR is to update the training script and to propose a collab training to make it accessible to everyone.

Please let me now in this PR if you have comment on the collab too : (https://colab.research.google.com/gist/MateoLostanlen/1300692a2ab41418276b455f4eeab64c/train-wildfire.ipynb)

### CODE REVIEW

**Comment by MateoLostanlen** [comment]
https://github.com/frgfm/Holocron/pull/118 needs to be merge before this PR :)

File: references/classification/WildFire/torch/train.py [hunk 1]
```diff
@@ -5,303 +5,151 @@
...
+    start_time = time.time()
+    trainer.fit_n_epochs(args.epochs, args.lr, args.freeze_until)
+    total_time_str = str(datetime.timedelta(seconds=int(time.time() - start_time)))
+    print('Training time {}'.format(total_time_str))
+
+
+def parse_args():
     import argparse
-    parser = argparse.ArgumentParser(description='PyroNear Classification Training',
+    parser = argparse.ArgumentParser(description='Holocron Classification Training',
```
Comment (frgfm) [inline references/classification/WildFire/torch/train.py]: little typo here ;)

File: references/classification/WildFire/torch/train.py [hunk 2]
```diff
@@ -5,303 +5,151 @@
...
-            val_loss += criterion(outputs, target).item()
-            # Index of max log-probability
-            pred = outputs.max(1, keepdim=True)[1]
-            correct += pred.eq(target.view_as(pred)).sum().item()
-            targets += x.size(0)
-    val_loss /= len(test_loader)
-    acc = correct / targets
-
-    return val_loss, acc
+#!/usr/bin/env python
```
Comment (frgfm) [inline references/classification/WildFire/torch/train.py]: no need for this

File: references/classification/WildFire/torch/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Train WildFire
+
+We propose here a demonstration of how to use pyro-vision to train a model to detect wildfire outbreaks.
+
+This script allow to train a binary classification model on our WilfFire dataset. You can train here all models availables on Holocron repository:
```
Comment (frgfm) [inline references/classification/WildFire/torch/README.md]: "allows you to train" and "available"

Comment (frgfm) [reply references/classification/WildFire/torch/README.md]: Also, you can put the URL directly on the word "Holocron"

File: references/classification/WildFire/torch/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+https://github.com/frgfm/Holocron/tree/master/holocron/models
+
+You can either follow the procedure below to train the model on your own machine or click on the following collab link to use our training notebook  on powerful and free machine.
```
Comment (frgfm) [inline references/classification/WildFire/torch/README.md]: double space after "notebook"
and "on powerful & free machine" --> "on remote servers for free"

File: references/classification/WildFire/torch/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/gist/MateoLostanlen/1300692a2ab41418276b455f4eeab64c/train-wildfire.ipynb)
+
+## Install
+
+Install pyro-vision first
+
+```bash
+pip install pyronear
```
Comment (frgfm) [inline references/classification/WildFire/torch/README.md:17]: This will need to be changed after next bump on pypi

**Review by frgfm — COMMENTED** [review]
Thanks for the PR! I left a few comments, let me know what you think

**Comment by MateoLostanlen** [comment]
Done :)

**Review by frgfm — APPROVED** [review]
Thanks for the PR! Also I'm wondering, but here we modified the training script for Wildfire. If this is gonna remain our video dataset, we need to find another name to specify that it's image classification :man_shrugging:

**Pull request merged**